cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaListenerReconcilerIngressV1Beta1Test.java

```diff
@@ -227,17 +129,14 @@ public MockKafkaListenersReconciler(
                 SecretOperator secretOperator,
                 ServiceOperator serviceOperator,
                 RouteOperator routeOperator,
-                IngressOperator ingressOperator,
-                IngressV1Beta1Operator ingressV1Beta1Operator) {
-            super(reconciliation, kafka, null, pfa, 300_000L, secretOperator, serviceOperator, routeOperator, ingressOperator, ingressV1Beta1Operator);
+                IngressOperator ingressOperator) {
+            super(reconciliation, kafka, null, pfa, 300_000L, secretOperator, serviceOperator, routeOperator, ingressOperator);
         }
 
         @Override
         public Future<ReconciliationResult> reconcile()  {
             return ingresses()
-                    .compose(i -> ingressesV1Beta1())
```

Comment:
I think we need to think about this test class. Its main purpose is that with Kube 1.16 it uses the v1beta1 methods and with Kube 1.22 it is using the v1 methods. It does not seem to check anything else. After removing the v1beta1 support, I don't think the v1 test makes any sense now. So maybe we should remove it completely?